cmd/kured/main.go

```diff
@@ -38,7 +38,11 @@ var (
 	version = "unreleased"
 
 	// Command line flags
+	forceDrain                bool
+	forceReboot               bool
+	forceTimeout              time.Duration
```

Comment:
My 2¢ are that a more explicit "drainTimeout" option makes the most sense here. That way it can be used more generally, whether or not folks want to force a reboot if the timeout expires.

Reply:
Good idea.